news.models.js

```diff
@@ -1,10 +1,15 @@
 const db = require("./db/connection.js");
 const format = require("pg-format");
+const endpointsJson = require("./endpoints.json");
 
 const readTopics = () => {
   return db.query("SELECT * FROM topics").then(({ rows }) => {
     return rows;
   });
 };
 
-module.exports = { readTopics };
+const readEndpoints = () => {
```

Comment:
Model is not needed in this case. You can deal with reading and sending back the endPoints.json in the controller.